src/atoms/icons/icons.module.scss

```diff
@@ -0,0 +1,15 @@
+@import 'src/assets/stylesheets/global';
```

Comment:
might have been a weird merge with Josh's PR but this has ended up in `/icons` instead of `/Icon`